src/components/organisms/NewProfileModal/components/ProfileModalQuestions/ProfileModalQuestions.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../profileModalStyles";
+
+$question-container-gap: 0.7rem;
+$question-answer-gap: 4px;
+$question-answer-gap-editable: 0.3rem;
```

Comment:
pixels

Reply:
Change rems to pixels or pixel to rems? Also: https://github.com/sparkletown/sparkle/pull/1934#discussion_r687462403

Reply:
rems to pixels. Answered in that comment.